### ISSUE TITLE
hi :)

### ISSUE THREAD

**Issue by zfogg** [Shraefkt/Terminal-video#1]
hi there! i see you starred my project, zfogg/ascii-chat ... i was just wondering what you're making here. it seems like an ascii video renderer for a terminal environment. is that right? super cool :D

**Comment by Shraefkt** [comment]
Yeah exactly, I thought it would be cool if it existed so I tried to make one, but I found yours so I’m currently trying it out :) thanks for the repo, it’s really awesome!

**Comment by zfogg** [comment]
amazing!! did you get my code working? it actually doesn't compile anymore for me lol.. opencv changed maybe? it segfaults ugh

**Comment by Shraefkt** [comment]
Unfortunately no, I tried to make some tweaks but it didn’t work. Anyways the code is nice to read and learn from
